nvm.sh

```diff
@@ -3573,16 +3584,25 @@ nvm() {
         >&2 nvm --help
         return 127
       fi
-
+      local NVMRC_SAVE_TEXT
+      NVMRC_SAVE_TEXT=''
+      if [ "${NVM_WRITE_TO_NVMRC}" -eq 1 ] && [ "${VERSION}" != '∞' ] && [ "${VERSION}" != 'N/A' ]; then
+        echo "$VERSION" | tee "${PWD}"/.nvmrc > /dev/null || {
+          nvm_err "Unable to write version to .nvmrc, aborted."
+          exit 3
+        }
+        NVMRC_SAVE_TEXT=" (saved to .nvmrc)"
+      fi
       if [ "_${VERSION}" = '_system' ]; then
         if nvm_has_system_node && nvm deactivate "${NVM_SILENT_ARG-}" >/dev/null 2>&1; then
           if [ "${NVM_SILENT:-0}" -ne 1 ]; then
-            nvm_echo "Now using system version of node: $(node -v 2>/dev/null)$(nvm_print_npm_version)"
+            nvm_echo "Now using system version of node:
+            $(node -v 2>/dev/null)$(nvm_print_npm_version)${NVMRC_SAVE_TEXT}"
```

Comment:
```suggestion
            nvm_echo "Now using system version of node: $(node -v 2>/dev/null)$(nvm_print_npm_version)${NVMRC_SAVE_TEXT}"
```
i don't think we want the node version on a separate line, that encourages people to parse it out of the message :-)